apps/admin-web/src/pages/api/Test.tsx

```diff
@@ -0,0 +1,16 @@
+import { useGetPendingReviewsQuery } from '@cgr/codegen'
+// import { HighlightHTML } from '@admin-web/common/HighlightHTML'
+import DOMPurify from 'isomorphic-dompurify'
+
+export default function Test() {
+  const reviewQuery = useGetPendingReviewsQuery()
+  return (
+    <>
+      Reviews:
+      {reviewQuery.data?.pendingReviews.map((data) => (
+        // Todo: change p tag to highlightHtml
+        <p dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(data.content ?? '') }} />
```

Comment:
Should this be inside HighlightHTML?

Reply:
And why does this page in api folder na?

Reply:
> Should this be inside HighlightHTML?

As HighlightHTML requires mui, but we have not installed and setup it yet. So, I use p tag for the time being.

Reply:
> And why does this page in api folder na?

I will migrate soon kub, sorry.